lib/DDGC/DB/Result/InstantAnswer.pm

```diff
@@ -395,6 +395,53 @@ many_to_many 'users', 'instant_answer_users', 'user';
 has_many 'instant_answer_topics', 'DDGC::DB::Result::InstantAnswer::Topics', 'instant_answer_id';
 many_to_many 'topics', 'instant_answer_topics', 'topic';
 
+after insert => sub {
+    my ( $self ) = @_;
+    my $schema = $self->result_source->schema;
+    $schema->resultset('ActivityFeed')->create(
+        $self->ddgc->config->subscriptions->created_ia_page( {
+            meta1        => $self->id,
+            meta2        => join('', map { sprintf ':%s:', $_ }
+                $self->topics->columns([qw/ name /])->all),
```

Comment:
```
:$topic::$topic::$topic:
```
huh.  what is logic behind this?  So you can grep for `:$topic:`?  Thats clever.  Its kind of a mouthful to type though.  Maybe you could make this a helper method on the `$self->topics` object.  related: The pod in `DDGC::Schema::Result::ActivityFeed` for this is different.